cmd/test.go

```diff
@@ -284,7 +284,10 @@ func (t *Test) GetCmd() *cobra.Command {
 				TestsetNoise:       testsetNoise,
 				WithCoverage:       withCoverage,
 				CoverageReportPath: coverageReportPath,
-			}, enableTele)
+			}, enableTele) {
+				t.logger.Error("failed to run the test")
```

Comment:
The log statement is not descriptive. Since, the test run has been completed successfully. And the result is also logged by the Test function only. So here we dont need to log. But please verify that the process exits with non zero if error is returned by the cmd handler funtion. Or you can use os.exit function here.